packages/cli-repl/src/arg-parser.ts

```diff
@@ -24,6 +24,7 @@ const OPTIONS = {
     'awsSessionToken',
     'awsIamSessionToken',
     'csfleLibraryPath',
+    'cryptLibraryPath',
```

Comment:
I think I’ll need to wait for confirmation on the new driver option name (if it is being changed) here